DEVELOP.rst

```diff
@@ -19,6 +19,7 @@ Installing node locally::
 
 Install the package dependencies::
 
+    $ export PATH="$PATH:/<absolute-path>/crate-admin/env/bin"
```

Comment:
The changes here shouldn't be necessary. Maybe what is missing is to activate the virtualenv properly. Though since the calls are made directly via `env/bin/...` that also shouldn't be necessary.

Reply:
reverted